packages/circuit-ui/components/Modal/Modal.module.css

```
@@ -1,11 +1,28 @@
-.base {
+.dialog {
   position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 0;
+  margin: auto;
+  overflow-y: auto;
+  pointer-events: none;
+  visibility: hidden;
   background-color: var(--cui-bg-elevated);
+  border: none;
+  border-radius: var(--cui-border-radius-mega);
   outline: none;
+  animation: fade-out 0.3s forwards;
 }
 
-.base::after {
-  position: fixed;
+.dialog.show {
+  pointer-events: auto;
+  animation: fade-in 0.3s forwards;
+}
+
+.dialog::after {
+  position: absolute;
```

This needs to be fixed positioned, otherwise it scrolls with the content.

```suggestion
  position: fixed;
```

However, this will position it relative to the viewport. Previously, the `transform` on the `dialog` ensured it was positioned relative to its parent. You'll need to replicate that somehow.

---

Clever use of `sticky`! 💡